kolibri/utils/options.py

```diff
@@ -352,3 +352,24 @@ def update_options_file(section, key, value, KOLIBRI_HOME, ini_filename="options
             file=conf.filename
         )
     )
+
+
+def generate_empy_options_file(options_path, options_data):
+
+    exclude_key = ["PICKLE_PROTOCOL"]
```

Comment:
> For example, I don't think we want to document the python pickle version?

Not sure if I understand what @indirectlylit's suggestion correctly - I feel like we need to remove the entire block of `Python` (since it only has one field `PICKLE_PROTOCOL`?) Otherwise users may get confused with the block `# [Python]` alone in options.ini file.